met-web/src/components/engagement/dashboard/report/EngagementDashboard.tsx

```diff
@@ -24,9 +24,9 @@ export const EngagementDashboard = () => {
     const [engagement, setEngagement] = useState<Engagement>(createDefaultEngagement());
     const [isEngagementLoading, setEngagementLoading] = useState(true);
 
-    const validateEngagement = (engagementToValidate: Engagement) => {
+    const failIfInvalidEngagement = (engagementToValidate: Engagement) => {
```

Comment:
I'm being picky with the names here but this seems a bit too descriptive, i'm sure its fine

Reply:
yeah I understand, the reason was validateEngagement was confusing and prompted a discussion with Saravan because it could mean validate user data, but here it is validating business data. This method is throwing an error when it fails so I thought, with Saravan's advice, to give it a fatal name. I thought of this.

Reply:
I will keep it in mind to improve on the naming and definitely value suggestions